routes/archive.js

```diff
@@ -4,7 +4,7 @@ const getArchiveAndChildren = require('../lib/get-archive-and-children');
 module.exports = (elastic, config) => ({
   method: 'GET',
   path: '/documents/{id}/{slug?}',
-  handler: (request, reply) => reply(),
+  handler: (request, reply) => HTMLResponse(request, reply, elastic, config),
```

Comment:
So if hapi-negotiator doesn't manage to match the media type this will execute by default, that's right? I try to find if hapi-negotiator has a default handler if none of the media type is matched but couldn't find this option yet. Do you know if it exists? Otherwise it's ok!